panel/1.0/getting_started.md

```diff
@@ -206,7 +210,7 @@ If you are not using `redis` for anything you should remove the `After=` line, o
 when the service starts.
 :::
 
-If you are are using redis for your system, you will want to make sure to enable that it will start on boot. You can do that by running the following command: 
+If you are are using redis for your system, you will want to make sure to enable that it will start on boot. You can do that by running the following command:
```

Comment:
While you're at it, maybe fix the double `are` that has been here for a while since making a PR for that one typo feels strange